tests/test_ocs_368.py

```diff
@@ -0,0 +1,78 @@
+"""
+Test OCS-368
+"""
+import logging
+import subprocess
+import tempfile
+import json
+import shlex
+from ocs import ocp
+from ocs.volumes import PVC
+
+logger = logging.getLogger(__name__)
+
+STORAGE_CLASS = 'rook-ceph-block'
+
+
+def run_oc_command(cmd):
+    """
+    Run an oc command and extract json output.
+
+    This uses temp files because pipes were breaking (please help me reviewers).
+
+    Args:
+        cmd (str): oc command to be run
+
+    Returns:
+        odata (dict): json information returned from the command run.
+
+    """
+    l_cmd = shlex.split(cmd + " -o json")
+    tfile = tempfile.mkstemp()
+    with open(tfile[1], 'w') as ofile:
+        subprocess.call(l_cmd, stdout=ofile)
+    with open(tfile[1]) as ifile:
+        odata = json.load(ifile)
+    return odata
+
+
+def is_pvc_bound(namespace):
+    """
+    Look at pvc created for this namespace and check if it is bound.
+
+    Args:
+        namespace (str): namespace being checked
+
+    Returns:
+        True if pvc is bound, False if not.
+
+    """
+    cmd = f"oc -n {namespace} get pvc"
+    odata = run_oc_command(cmd)
+    return odata['items'][0]['status']['phase'] == 'Bound'
+
+
+def run(namespace_name):
```

Comment:
I'm not sure what this function is needed for.
We moved to pytest few weeks ago.
Have you seen the last few live demos?